### ISSUE TITLE
Autofix alphabet order on operation error

### ISSUE THREAD

**Issue by davidNHK** [B2o5T/graphql-eslint#978]
**Describe the bug**
When autofix alphabet parameter it only able to sort parameter name

**To Reproduce**

Give 

```js
export const UPLOAD_SELFIE = gql`
  mutation uploadSelfie($file: Upload!, $data: SelfieInput!) {
    uploadSelfie(data: $data, file: $file) {
      filename
      id
      url
    }
  }
`;
```

**Expected behavior**

```js
export const UPLOAD_SELFIE = gql`
  mutation uploadMemberSelfie($data: SelfieInput!, $file: Upload!) {
    uploadSelfie(data: $data, file: $file) {
      filename
      id
      url
    }
  }
`;
```
**Actual Result**
```js
export const UPLOAD_SELFIE = gql`
  mutation uploadSelfie($data: Upload!, $file: SelfieInput!) {
    uploadSelfie(data: $data, file: $file) {
      filename
      id
      url
    }
  }
`;
```

**Environment:**

- OS:
- `@graphql-eslint/...`:
- NodeJS:

**Additional context**

**Comment by davidNHK** [comment]
It not a big deal if you covered you code with test coz it would prompt you error immediately 

if no test covered then i think you would need turn off the alphabet rules meanwhile

**Comment by B2o5T** [comment]
fixed in [@graphql-eslint/eslint-plugin@3.9.0](https://github.com/B2o5T/graphql-eslint/releases/tag/%40graphql-eslint%2Feslint-plugin%403.9.0) 🚀
